src/main/java/frc/robot/Robot.java

```diff
@@ -32,18 +32,39 @@ public void robotInit() {
         shooterSystem = new ShooterSystem();
         intakeSystem = new IntakeSystem();
         armSystem = new ArmSystem();
-        xboxController = new XboxController(0);
+        xboxController = new XboxController(RobotMap.XBOX_CONTROLLER_PORT);
 
-        POVButton dPadUp = new POVButton(xboxController, 0);
-        POVButton dPadDown = new POVButton(xboxController, 180);
+        armSystem.setDefaultCommand(
```

Comment:
So basically you are always running a motion to the shooter, and while doing so you are moving the intake.
why is this a default command?

Reply:
This is created for the purpose to simplify other code instead of write this part of code in other command group.

Reply:
I see, but what will realistically happen is that this command starts running, moves the arm to shooter and then ends. Afterwards it will start again (because no command is running on the arm) and will start moving and stop a few milliseconds after, and so on and so on. We might need a different solution

Reply:
But the default command doesn't end. He wait that ArmMoveToShooterCommand end, but he doesn't have 'isFinished' so he going to work for endless

Reply:
my mistake